tests/alive-tv/memory/memcmp-ub.src.ll

```diff
@@ -0,0 +1,58 @@
+target datalayout = "e-p:64:64:64"
+
+define i32 @ub() {
+  %p = alloca i32
+  %p8 = bitcast i32* %p to i8*
+  %res = call i32 @memcmp(i8* %p8, i8* %p8, i64 4) ; ub
```

Comment:
why is this UB? The input has 4 bytes.

Reply:
Because it is comparing poison bytes. If memcmp is converted into a loop, it has branch on poison, so this is UB.